tests/kernel/timer/timer_error_case/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  kernel.timer:
+    tags: kernel timer userspace ignore_faults
+    platform_exclude: qemu_x86_coverage qemu_arc_em qemu_arc_hs
```

Comment:
reason why those are being skipped?

Reply:
exclude those platform because until "icount" is not implemented in QEMU for ARC, but I realized those cases just for error condition, so remove this.